tetrateci/create_istio_release.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir /tmp/istio-release
+sudo gem install fpm
+sudo apt-get install go-bindata -y
+envsubst < ./tetrateci/manifest.yaml.in > manifest.yaml
+git clone https://github.com/istio/release-builder --depth=1
+cd release-builder
+go run main.go build --manifest ../manifest.yaml
+go run main.go publish --release /tmp/istio-release/out --dockerhub $HUB
```

Comment:
Why aren't we running the release validate?

Reply:
Ahh, missed that